### PR TITLE
[ADMIN][ADI] Add Idea admin

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Repository/AdherentRepository.php [hunk 1]
```diff
@@ -750,4 +752,22 @@ public function refresh(Adherent $adherent): void
     {
         $this->getEntityManager()->refresh($adherent);
     }
+
+    public function getIdeaContributors(Idea $idea): array
```
Comment (JeremieSamson) [inline src/Repository/AdherentRepository.php]: isn't it the same as `IdeaRepository` ?

File: templates/admin/ideas_workshop/idea_contributors.html.twig [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{% extends 'SonataAdminBundle::standard_layout.html.twig' %}
+
+{% block sonata_page_content_header %}
+    <div class="sticky-wrapper">
+        <nav class="navbar navbar-default" role="navigation" style="width: auto;">
+            <div class="container-fluid">
+                <div class="navbar-header">
+                    <a class="navbar-brand" href="#">
+                        Liste des contributors de l'idée "{{ idea.name }}"
```
Comment (Remg) [inline templates/admin/ideas_workshop/idea_contributors.html.twig]: Liste des contributeurs*

File: templates/admin/list/list_author.html.twig [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
+
+{% block field %}
+    {% if author is not defined%}
```
Comment (Remg) [inline templates/admin/list/list_author.html.twig]: missing whitespace before closing tag

File: templates/admin/list/list_author.html.twig [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        {% set author = object.author %}
+    {% endif %}
+
+    {% if author %}
+        <a href="{{ path('admin_app_adherent_show', {'id': author.id}) }}">
```
Comment (Remg) [inline templates/admin/list/list_author.html.twig]: `{ id: author.id }`

File: templates/admin/ideas_workshop/list_show_page.html.twig [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{% if object.isPending or object.isFinalized %}
+    <a href="{{ path('react_app_ideas_workshop_note', {id: object.uuid} ) }}" class="btn btn-sm btn-default view_link"
```
Comment (Remg) [inline templates/admin/ideas_workshop/list_show_page.html.twig]: `{ id: object.uuid }` (add spaces)

File: src/Repository/AdherentRepository.php (hunk 1 above)
Comment (illusionOfParadise) [reply src/Repository/AdherentRepository.php]: I removed it.

**Pull request merged**